test/data/repositories/rhel-85.json

```diff
@@ -0,0 +1,19 @@
+{
+  "x86_64": [
+    {
+      "name": "baseos",
+      "baseurl": "http://download.eng.bos.redhat.com/rhel-8/development/RHEL-8/latest-RHEL-8.5.0/compose/BaseOS/x86_64/os/",
```

Comment:
Why don't we use rpmrepo?

Reply:
Just a note, that we actually never used it for unreleased RHEL versions. We do the same for 8.4...

Reply:
Like @thozza said.  I went with what we used for 8.4.

Reply:
Ah, I guess it makes sense. Let's change it if it makes us some troubles.